src/components/Header/Header.tsx

```diff
@@ -0,0 +1,21 @@
+export const Header: React.FC = () => {
+  return (
```

Comment:
Destruct `FC` from 'react' and try to avoid `return` when you haven't any logic

```
export const Header: FC = () => (
  <header className="todoapp__header">
```